goodmap/data_models/location.py

```
@@ -156,7 +186,7 @@ def _build_field_definition(
 
 def create_location_model(
     obligatory_fields: list[tuple[str, str]] | list[tuple[str, Type[Any]]],
-    categories: dict[str, list[str]] | None = None,
+    categories: dict[str, list[str]] = {},
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)